content/PDOStatementExecute/PDOStatementExecute.facts.md

```diff
@@ -0,0 +1,51 @@
+### Facts for Prepared Statement with Binding Methods
```

Comment:
Please update also the names of the files, directories, and summary.md based on the changes that we have in our lesson title.